aspnetcore/fundamentals/minimal-apis/aspnetcore-openapi.md

```diff
@@ -161,6 +161,17 @@ builder.Services.AddOpenApi(options =>
 })
 ```
 
+## Customizing OpenAPI endpoints with endpoint metadata
```

Comment:
This content is great, but I would have placed it earlier in this doc. I think it could be just above "Options for customizing OpenAPI document generation".

Reply:
If @captainsafia  agrees, I'll fix this in https://github.com/dotnet/AspNetCore.Docs/pull#32626